test/PhysicsPluginsList.hh

```diff
@@ -21,18 +21,54 @@
 #include <string>
 #include <vector>
 
+#include <ignition/common/Filesystem.hh>
+#include <ignition/common/Util.hh>
+
 #ifndef dartsim_plugin_LIB
-#define dartsim_plugin_LIB ""
+#define dartsim_plugin_LIB "__main__/ign_physics/dartsim/libignition-physics-dartsim-plugin.so"
 #endif
 
 #ifndef bullet_plugin_LIB
 #define bullet_plugin_LIB ""
 #endif
 
 #ifndef tpe_plugin_LIB
-#define tpe_plugin_LIB ""
+#define tpe_plugin_LIB "__main__/ign_physics/tpe/libignition-physics-tpe-plugin.so"
 #endif
 
+/////////////////////////////////////////////////
+std::string resolveLibrary(const std::string &_path)
+{
+  auto it = _path.find("__main__");
```

Comment:
turns out our internal version of Bazel does not use `__main__` in its paths. could we only check the existence of `TEST_SRCDIR`?